src/Component/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Comments } from '../../Types';
+import { CommentInfo } from '../CommentInfo';
+import './Style.css';
+
+type Props = {
+  comments: Comments[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <div>
+      <ul className="list-comment">
+        <span className="title">Comments</span>
```

Comment:
```suggestion
      <span className="title">Comments</span>
      <ul className="list-comment">  
```
Перенеси цей спан вище ul. Такий код не валідний, бо в ul мають міститись li, а не якісь інші теги